src/ntrip_client/ntrip_client.py

```diff
@@ -195,7 +196,7 @@ def reconnect(self):
           self._logerr('Reconnect to http://{}:{} failed. Retrying in {} seconds'.format(self._host, self._port, self.reconnect_attempt_wait_seconds))
           time.sleep(self.reconnect_attempt_wait_seconds)
         elif self._reconnect_attempt_count >= self.reconnect_attempt_max:
-          self._reconnect_attempt_count = 0
+          # self._reconnect_attempt_count = 0 #this hides the retry count from the excepton?
```

Comment:
Yup this was a mistake. Just put this after the log, and that should be fine.

Realistically, it is probably fine to just not set it, but the `ntrip_client.py` is meant to be agnostic of how ROS runs it, so it is possible that someone could catch that exception and want the value to be updated properly